docs/get_started/index.md

```diff
@@ -174,45 +105,149 @@ AWS images with MXNet installed:
 
 <div class="docker">
 
-<h3>Python</h3>
+Pre-build docker images are available at [docker hub](https://hub.docker.com/r/mxnet/).
+
+<div class="python">
 
 ```bash
 docker pull mxnet/python
 docker pull mxnet/python:gpu
 ```
 
-<h3>Scala</h3>
+</div> <!-- python -->
+
+<div class="scala">
 
 ```bash
 docker pull mxnet/scala
 ```
 
-<h3>R</h3>
+</div> <!-- scala -->
+
+<div class="r">
 
 ```bash
 docker pull mxnet/r-lang
 docker pull mxnet/r-lang:gpu
 ```
 
-<h3>Julia</h3>
+</div> <!-- r -->
+
+<div class="julia">
 
 ```bash
 docker pull mxnet/julia
 docker pull mxnet/julia:gpu
 ```
 
+</div> <!-- julia -->
+
 Refer to [docker/](../../docker/) for more details.
 
 </div> <!-- docker -->
 
 <div class="build-from-source">
 
-Refer to [setup](./setup.md) for details on building MXNet from source codes for
-various systems.
+Refer to the [building from source document](./build_from_source.md) for details
+on building MXNet from source codes for various systems.
 
 </div> <!-- build-from-source -->
 
-<script type="text/javascript" src='../../_static/js/options.js'></script>
+
+## Quick Overview
+
+MXNet provides an imperative *n*-dimensional array interface:
+
+```python
+>>> import mxnet as mx
+>>> a = mx.nd.ones((2, 3))
+>>> b = a * 2 + 1
+>>> b.asnumpy()  # print b by converting to a numpy.ndarray object
+array([[ 3.,  3.,  3.],
+       [ 3.,  3.,  3.]], dtype=float32)
+```
+
+```scala
+scala> import ml.dmlc.mxnet._
+import ml.dmlc.mxnet._
+
+scala> val arr = NDArray.ones(2, 3)
+arr: ml.dmlc.mxnet.NDArray = ml.dmlc.mxnet.NDArray@f5e74790
+
+scala> (arr * 2 + 1).toArray
+res0: Array[Float] = Array(3.0, 3.0, 3.0, 3.0, 3.0, 3.0)
+```
+
+```r
+> require(mxnet)
+Loading required package: mxnet
+> a <- mx.nd.ones(c(2,3))
+> a * 2 + 1
+     [,1] [,2] [,3]
+[1,]    3    3    3
+[2,]    3    3    3
+```
+
+```julia
+julia> using MXNet
+
+julia> a = mx.ones((2,3))
+mx.NDArray{Float32}(2,3)
+
+julia> Array{Float32}(a * 2 + 1)
+2×3 Array{Float32,2}:
+ 3.0  3.0  3.0
+ 3.0  3.0  3.0
+```
+
+```perl
+pdl> use AI::MXNet qw(mx)
+pdl> $a = mx->nd->ones([2, 3], ctx => mx->gpu())
```

Comment:
i don't have perl installed, so i don't want to make change unless i can test it.